inst/scripts/importPDB.R

```diff
@@ -0,0 +1,158 @@
+# importPDB.R
+# Non-exported function
+
+#=============================STOP & SAVE======================================
+# Provide a mechanism to stop:
+# REFERENCE:
+#   http://stackoverflow.com/questions/38755283/break-loop-with-keyboard-input-r
+# requires library(tcltk2)
+win1 <- tktoplevel()
+butStop <- tkbutton(win1, text = "Stop",
+                    command = function() {
+                        assign("stoploop", TRUE, envir = .GlobalEnv)
+                        tkdestroy(win1)
+                    })
+
+butSave <- tkbutton(win1, text = "Pause & Save",
+                    command = function() {
+                        assign("stoploop", TRUE, envir = .GlobalEnv)
+                        print("Finishing the current gene..")
+                        saveRDS(genes, file = "inst/tmp/genes.rds")
+                        saveRDS(GeneData, "inst/extdata/GeneData.rds")
+                        tkdestroy(win1)
+                    })
+tkgrid(butStop)
+tkgrid(butSave)
+stoploop <- FALSE
+
+readRDS(MT.rds)
+
+#=============================(LOAD & CONTINUE)/START==========================
+# Check if there was a saved progress from before.
+if (file.exists("inst/tmp/genes.rds")){

```

Comment:
It's not a good idea to hardcode a file-path like that. Use a variable ```PROGRESSFILE``` that you define at the top of the script, and **tell the user** that you are doing this. Worst case scenario: the user forgot about the progress file for a different project and your code silently patches two incompatible lists together.